pocs/tests/utils/test_utils.py

```diff
@@ -12,6 +12,7 @@
 from pocs.utils import CountdownTimer
 from pocs.utils import error
 from pocs.camera import list_connected_cameras
+from pocs.utils.library import load_library
```

Comment:
```suggestion
from pocs.utils.library import load_library as load_c_library
```

I'm opting to change this here to keep consistency with what the name will be in `panoptes-utils` but feel free to reject this change and change line 47 instead.